Resources/Prototypes/Catalog/uplink_catalog.yml

```
@@ -93,7 +93,7 @@
   description: uplink-explosive-grenade-desc
   productEntity: ExGrenade
   cost:
-    Telecrystal: 4
+    Telecrystal: 2
```

```suggestion
    Telecrystal: 4
```

Thrown explosives should still be expensive.

---

So we don't spam as much explosions.